README.rst

```diff
@@ -745,6 +745,29 @@ setting ``REDIS_CLIENT_KWARGS``.
         }
     }
 
+
+Closing Connections
+~~~~~~~~~~~~~~~~~~~
+
+django-redis closes all the connections to Redis server by default, irrespectively of its current usage.
+
+You can change this behaviour for all caches by setting ``DJANGO_REDIS_CLOSE_CONNECTION = False``
+in the django settings or at cache level by setting ``CLOSE_CONNECTION: False`` in the OPTIONS
```

Comment:
It is still too early to set it as true by default

Reply:
done